flagging_site/blueprints/flagging.py

```diff
@@ -60,8 +62,7 @@ def index() -> str:
     }
     return render_template('index.html', flags=flags)
 
-
-@bp.route('/output_model')
+@bp.route('/output_model', methods = ["GET"])
```

Comment:
PEP 8 convention for passing kwargs is to not use a space between the assigned value and the variable. Also, let's make sure we are consistent with use of single or double quotes across our code.

So this line should be the following:

```python
@bp.route('/output_model', methods=['GET'])
```